recipes/visp/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "visp" %}
+{% set version = "3.5.0" %}
+{% set major_minor_version = version.split('.')[0] + version.split('.')[1] + version.split('.')[2] %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lagadic/visp/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 494a648b2570da2a200ba326ed61a14e785eb9ee08ef12d3ad178b2f384d3d30
+  patches:
+    - patches/0001-Fix-invalid-usage-of-OGRE_DIR-in-CMakeLists.txt.patch
+    - patches/0002-Disable-AVX-and-AVX2-for-Simd-embedded-lib.patch  # [win]
```

Comment:
Could you document the reason to disable those instruction sets on Windows?

Reply:
I don't have the final word on what must be published on conda-forge but deactivating it while documenting the reason is reasonable and sufficient to me; this can be fixed with other builds or with another version.

Reply:
Fine. Thanks for your feedback.